doc/sphinx/proofs/writing-proofs/rewriting.rst

```diff
@@ -662,7 +662,7 @@ the conversion in hypotheses :n:`{+ @ident}`.
    This applies :tacn:`simpl` only to the :n:`{+ @natural}` applicative subterms whose
    head occurrence is :n:`@qualid` (or :n:`@string`).
 
-.. flag:: Debug RAKAM
+.. flag:: Debug "RAKAM"
```

Comment:
This is no longer an independent flag, right?  So "RAKAM" should be described under the `Debug` flag.  Also, how will the following flags change?

```
  Debug Auto
  Debug Cbv  
  Debug Eauto
  Debug Ssreflect
  Debug SsrMatching
  Debug Tactic Unification
  Debug Trivial
  Debug Unification
```